source/vir/src/triggers_auto.rs

```diff
@@ -39,6 +39,7 @@ enum App {
     // u64 is an id, assigned via a simple counter
     Other(u64),
     VarAt(UniqueIdent, VarAt),
+    BitOp,
```

Comment:
This needs to track which bit vector operator is represented (`BitOp(...something...)`).  For example, xor can't be the same as shr here.

Reply:
I will add information on that. Thanks!